src/components/widgets/CheckBox.js

```diff
@@ -13,34 +13,41 @@ class CheckBox extends PureComponent {
     const { input } = this.props;
     if (typeof input !== 'undefined') {
       const { onChange } = input;
-      onChange(false);
+      onChange(' ');
     }
   }
 
   handleToggle(e) {
     const { input } = this.props;
     if (typeof input !== 'undefined') {
-      this.props.input.onChange(e.target.checked);
+      if (e.target.checked) {
+        this.props.input.onChange(e.target.value);
+      } else {
+        this.props.input.onChange(' ');
+      }
     }
   }
 
   render() {
-    const { title, input } = this.props;
+    const { title, input, options } = this.props;
```

Comment:
Let's rename "options" to "value".

Reply:
@mark `value` is an instance API of <Field /> already (https://redux-form.com/7.1.1/docs/api/field.md/), so changing that prop to value overwrites some-other thing and alters it's behaviour. I changed it to `checkBoxValue`, is that better?